spec/controllers/application_controller_spec.rb

```diff
@@ -18,18 +18,30 @@
       expect(application_controller.blackcat_message?(:readonly)).to be false
     end
 
-    it 'readonly is set to false by default if blackcat_messages.yml does not exists' do
+    it 'readonly is set to false by default if blackcat_messages.yml does not exist' do
       allow(application_controller).to receive(:message_file?).and_return(false)
       expect(application_controller.blackcat_message?(:readonly)).to be false
     end
 
-    it 'readonly is set to false if blackcat_messages.yml exists but read_only key is missing' do
+    it 'readonly is set to false if blackcat_messages.yml exists but readonly key is missing' do
       allow(application_controller).to receive(:message_file?).and_return(true)
       allow(application_controller).to receive(:message_status).and_return({})
       expect(application_controller.blackcat_message?(:readonly)).to be false
     end
   end
 
+  describe 'alert message' do
+    before do
+      allow(application_controller).to receive(:blackcat_message).with(:alert).and_return('An F5 Sharknado is approaching the eastern seaboard, take cover.')
+      allow(application_controller).to receive(:blackcat_message?).with(:alert).and_return(true)
+    end
+
+    it 'flashes the alert message when present in blackcat_messages.yml' do
+      application_controller.send(:flash_alert)
+      expect(flash[:error]).to match('An F5 Sharknado is approaching the eastern seaboard, take cover.')
```

Comment:
🦈  😆

Reply:
Will we be prepared for this? I hope so!

Reply:
😬 👀